pkgs/by-name/lo/lovely-injector/package.nix

```
@@ -0,0 +1,39 @@
+{
+  fetchFromGitHub,
+  rustPlatform,
+  lib,
+}:
+let
+  version = "0.6.0";
+  lovelyInjector = fetchFromGitHub {
+    owner = "vgskye";
+    repo = "lovely-injector";
+    rev = "3224915f4d47b557c34b5012797cf92d4cc629af";
+    hash = "sha256-fzkuuu6pmvqeJa7qlX8jhtCLC4oYRLUm1hqHTRiYEX8=";
+  };
+in
+rustPlatform.buildRustPackage rec {
+  pname = "lovely-injector";
+  inherit version;
+  src = lovelyInjector;
+  useFetchCargoVendor = true;
+  cargoHash = "sha256-Mkmj+ENdUge1V1cVAQOV2K01sYKEyhxTse0f5o6H6Xc=";
+  # Disable tests, they are broken on my machine
+  doCheck = false;
```

```suggestion
  # no tests
  doCheck = false;
```
The only thing "tested" with cargo test is the constructor function that initializes lovely, and it doesn't work in the test suite as evidenced above.